src/faq.md

```diff
@@ -149,10 +149,10 @@ appreciation through careful design choices and fair communication about them.
 ### Q. Is Dart a statically typed language?
 
 Dart 1.x is statically typed if you opt in to [strong mode]
-and use a compiler (such as the Dart Development Compiler
-([dartdevc]({{site.webdev}}/tools/dartdevc))),
-that has both static and runtime checks. In this environment, Dart
-has a sound type system, which guarantees that an expression of one
+and use a compiler (such as the Dart Development Compiler, aka
+[dartdevc]({{site.webdev}}/tools/dartdevc)),
+that has both static and runtime checks. With both static and runtime checks,
```

Comment:
With -> When the tools have..., then

(I'm not crazy about repeating "both static and runtime checks", but I can't think of a better way to say it right now.)